tests/test_fakes.py

```diff
@@ -932,6 +932,34 @@ def test_show_tables(dcur: snowflake.connector.cursor.SnowflakeCursor):
     assert [r.name for r in dcur.description] == ["created_on", "name", "kind", "database_name", "schema_name"]
 
 
+def test_show_primary_keys(dcur: snowflake.connector.cursor.SnowflakeCursor):
+    dcur.execute("CREATE TABLE example (id int, name varchar, PRIMARY KEY (id, name))")
+
+    dcur.execute("show primary keys")
+    result = dcur.fetchall()
+
+    assert result == [
+        {
+            "constraint_name": "db1_schema1_example_pkey",
+            "table_name": "EXAMPLE",
+            "column_name": "ID",
+        },
+        {
+            "constraint_name": "db1_schema1_example_pkey",
+            "table_name": "EXAMPLE",
+            "column_name": "NAME",
+        },
+    ]
+
+    dcur.execute("show primary keys in schema1")
+    result2 = dcur.fetchall()
+    assert result == result2
+
+    dcur.execute("show primary keys in information_schema")
```

Comment:
it's not, I mostly just added it to prove that the `IN SCHEMA` filter was properly scoping the the schema. Otherwise the above two assertions could both just be the current databases' total set of primary keys.